Jenkinsfile

```diff
@@ -1,5 +1,10 @@
-mdsdToolsEclipsePipeline {  
-    webserverDir = 'thirdparty-library'
-    updateSiteLocation = 'target/repository'
-    skipCodeQuality = true
-} 
\ No newline at end of file
+@Library('MDSDtoolsLib@master') _
```

Comment:
We do not need this annotation, do we? The only downside of having it there is that we cannot rename the library name on our build server independently.

Reply:
True.